packages/react/src/utils/dom.ts

```diff
@@ -131,7 +131,7 @@ export function createFocusableRef<T extends HTMLElement = HTMLElement>(
 }
 
 export function useDOMRef<T extends HTMLElement = HTMLElement>(
-  ref: RefObject<T | null> | Ref<T | null>
+  ref?: RefObject<T | null> | Ref<T | null>
```

Comment:
Can this be optional @jrgarciadev? I think it could be because of optional refs passed as props.